system-tests/driver-config/jenkins.sh

```diff
@@ -29,7 +29,7 @@ targets:
       PATH: "/usr/local/bin:$PATH"
 
     scripts:
-      proxy: http-server --proxy-secure=false -p 4201 -P \$CLUSTER_URL dist
+      proxy: http-server --proxy-secure=false -p 4201 -P \$CLUSTER_URL ../../dist
```

Comment:
The `dcos-system-test-driver` always ensures that the paths are relative to the file being processed. However we moved the file from the root directory (previously called `driver-config.sh`) into `system-tests/driver-config`.

We could however use a variable in the file, like `BASE_DIR=../..` and then use `$BASE_DIR}/dist` in the script :thinking: